plat/mediatek/mt8183/drivers/pmic/pmic_wrap_init.c

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
Isn't the pwrap/rtc interface 100% identical with MT8173? Please factor out code like this into plat/mediatek/common, don't duplicate it.